frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -268,6 +273,7 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
         openAsNewInsight: (tileId: TileId, tabId?: string) => {
             return { tileId, tabId }
         },
+        setConversionGoalWarning: (warning: ConversionGoalWarning | null) => ({ warning }),
```

Comment:
This is my lack of knowledge on kea showing up: why is it that we're using an object here, but then on line 466 we are doing it differently?

My guess is that whatever we pass in here is actually what we get as the second parameter on line 466. Is that right? What's the rationale here?

And yes, I know the answer is probably: Rafa, you have to learn Kea.